api/v1alpha1/url_shortener.proto

```diff
@@ -10,6 +10,10 @@ message ShortURLsRequest {
   string url = 1;
 }
 
+message ShortURLsRequests {
+  repeated ShortURLsRequest urls = 1;
```

Comment:
I wouldn't do this, instead, take a look that we need to pass a list of URLs directly, instead of a list of ShortURLRequest.
I would use `repeated string urls = 1;` instead, and call the message `BalanceURLsRequest`